2017/CG-05.md

```diff
@@ -98,6 +98,188 @@ Please register before the event [here](https://goo.gl/forms/LQVhuRuI22lK8Umw1).
 
 ## Proposal details
 
+### Threads proposal details
+
+#### i8 and i16 Value Types
+
+Tracking Issue: [i8/i16 values with limited operators #1049](https://github.com/WebAssembly/design/issues/1049)
+
+**Poll**:
+
+> Add `i8` and `i16` value types.
+
+Adding i8 and i16 value types can reduce the number of opcodes required to
+implement atomics. Because they were not considered for the initial WebAssembly
+design, adding them now introduces some potential inconsistencies.
+
+#### Encoding Scheme
+
+Tracking issue: [Encoding Scheme #12](https://github.com/WebAssembly/threads/issues/12)
+
+**Poll**:
+
+> 1. Use `0xf0` prefix byte for all threading opcodes.
+> 2. Use `0xff` prefix byte for all threading opcodes.
+
+This should be coordinated with the SIMD proposal.
+
+**Poll**:
+
+> 1. Encoding proposal #1: No new non-atomic operators. Add 107 new atomic operators.
+> 2. Encoding proposal #2: Use i8 and i16 value types. Add 12 new conversion operators. Add 40 new atomic operators.
+> 3. Encoding proposal #3: Add 5 new sign-extend operators. Add 67 new atomic operators.
+
+See the full description of these proposals in the [overview](https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md).
+
+#### Matching SharedArrayBuffer/Atomics for v1
+
+Tracking issue: [Matching behavior of SharedArrayBuffers in ES spec #5](https://github.com/WebAssembly/threads/issues/5)
+
+**Poll**:
+
+> Include the same atomic and threading primitives as provided by the ES spec.
+
+This includes:
+* `is_lock_free`
+* `wait` for 32-bit integers
+* `wake`
+* Atomic ops for 8-, 16-, and 32-bit signed and unsigned integers:
+  - `load` and `store`
+  - `add`, `sub`, `and`, `or`, `xor`, `xchg`
+  - `cmpxchg`
+
+#### Float atomics
+
+Tracking issue: [float atomics, i64 atomics, i64.wait #7](https://github.com/WebAssembly/threads/issues/7)
+
+**Poll**:
+
+> Include `f32` and `f64` atomic `load` and `store` operators.
+
+These are not currently included in the ES spec. Similar functionality is
+proposed for C++20.
```

Comment:
C++20 has more than what is proposed here: it has add and sub, and allows different rounding / subnormal / FP exceptions. C++11 already had FP load / store / xchg / cmpxchg.